platforms/unix/vm/sqUnixMain.c

```diff
@@ -863,13 +863,13 @@ sqInt  primitivePluginRequestState(void)	{ return dpy->primitivePluginRequestSta
 
 static void outOfMemory(void)
 {
-  /* pushing stderr outputs the error report on stderr instead of stdout */
+  /* pushing VM_ERR() outputs the error report on VM_ERR() instead of stdout */
```

Comment:
What is the significance of changing this comment?  (I'm not familiar with *pushOutputFile*)